quicklinks/quicklinks.json

```diff
@@ -0,0 +1,33 @@
+{                                                                                                          
```

Comment:
(nit) This file has lots of trailing spaces on every line. I assume they are not needed.